third_party/move/move-compiler-v2/tests/checking/typing/while_body_invalid.exp

```diff
@@ -0,0 +1,37 @@
+
+Diagnostics:
+error: `integer` not compatible with `()` in constant
```

Comment:
To clarify yet again, "in constant" is not useful here.  

Expected `()` for while loop body but found `bool` constant.

Reply:
And to say it again, you are commenting on code which was outdated some time ago.

Reply:
Sorry, yes.